src/ucp/proto/proto_multi.c

```diff
@@ -86,6 +99,15 @@ ucs_status_t ucp_proto_multi_init(const ucp_proto_multi_init_params_t *params,
         max_bandwidth = ucs_max(max_bandwidth, lane_perf->bandwidth);
     }
 
+    /* Sort lanes by bandwidth */
+    if (num_lanes > 1) {
+        /* If type of the first lane is not the same as in the middle, then
+         * exclude this lane from sorting */
```

Comment:
this comments just repeats what the code does. Can you pls descrive the reason for that? E.g. that when the first lane is different is should always be used first e.g. to preserve ordering with eager protocols for TAG and AM APIs.

Reply:
ok